releasenotes/notes/support_terra_primitive_022-8852b784608bcdcb.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
```

```suggestion
fixes:
```

This is a new feature of Terra, but it might be a bug fix for Aer.